packages/core/install-package/src/validateModuleSpecifiers.js

```diff
@@ -0,0 +1,12 @@
+// @flow
+
+const MODULE_REGEX = /((@[^/\s]+\/){0,1}([^/@\s]+)){1}(@[^/\s]+){0,1}/;
+
+export default function validateModuleSpecifiers(
+  modules: Array<string>
+): Array<string> {
+  return modules.map(module => {
+    // $FlowFixMe, not sure why this happens
+    return MODULE_REGEX.exec(module)[0] || '';
```

Comment:
Should this fallback to an empty string or throw an error, seems like this should always return a package name so this will probably never happen unless there's a bug somewhere.